src/app/bucket/bucket_file_reader.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+from google.cloud import storage
+from logging_config import logging
+
+logger = logging.getLogger(__name__)
+
+
+class BucketFileReader:
+    def __init__(self):
+        self.storage_client = storage.Client()
+
+    def get_file_from_bucket(self, filename: str, bucket_name: str):
+        """Used by the cloud function."""
+        bucket = self.storage_client.bucket(bucket_name)
+
+        try:
+            dataset = json.loads(bucket.blob(filename).download_as_string())
+            isValid, message = self.validate_keys(dataset)
+            if isValid is True:
+                return dataset
+            else:
+                logger.error(
+                    f"The mandatory key(s) {message} is/are missing in the JSON object."
+                )
+                return None
+        except ValueError as e:
+            logger.error(f"Invalid JSON in the file {filename} - %s" % e)
```

Comment:
very minor - could you use just one style of string fomatting here please

Reply:
Done